workflows/proteomics/clinicalmp/clinicalmp-database-generation/iwc-clinicalmp-database-generation.ga

```diff
@@ -0,0 +1,348 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Generating a customized database for clinical metaproteomics",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "identifier": "0000-0001-9818-0537",
+            "name": "Subina Mehta"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "release": "0.1",
+    "name": "iwc-clinicalmp-database-generation",
```

Comment:
The name is the name users see in the workflow list or in https://iwc.galaxyproject.org/, maybe something like
```suggestion
    "name": "Generate a Clinical Metaproteomics Database",
```
?

Reply:
I see you've marked this as resolved but didn't change the name, did you mean to do that ?